controller/api/models.py

```diff
@@ -820,12 +832,45 @@ class Domain(AuditedModel):
     owner = models.ForeignKey(settings.AUTH_USER_MODEL)
     app = models.ForeignKey('App')
     domain = models.TextField(blank=False, null=False, unique=True)
+    cert = models.ForeignKey('DomainCert', null=True)
 
     def __str__(self):
         return self.domain
 
 
 @python_2_unicode_compatible
+class DomainCert(AuditedModel):
```

Comment:
Why `DomainCert` and not `Cert` or `Certificate`?  We refer to them as "certs" all over the place.. the CLI, the URL patterns, the etcd keyspace -- as well as as when we talk about them generally.